### PR TITLE
Convert Ruby cheat sheets

### PR DESCRIPTION
**Pull request by nerdinand**
I took the cheat sheet apart into 3 parts:

- Integers, Floats, Strings
- Arrays
- Hashes

I noticed the Hashes part still uses strings as hash keys, not symbols. But I guess we can change that later...
Also, the layout breaks apart a little bit for these (too little horizontal space). Tell me what you think...

### CODE REVIEW

File: cheat_sheets/hashes.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+| Hash access        | Access a value by its key                                            | _hash[key]_                                   | `hash["key"]`                                                                        |
+| Key deletion       | Delete a key-value pair by its key                                   | _hash.delete(key)_                            | `hash.delete("key")`                                                                 |
+| Empty hash         | Remove all pairs from the hash                                       | _hash.clear_                                  | `hash.clear`                                                                         |
+| Iterate over hash  | Iterate over all the pairs in the hash                               | _hash.each do \|key, value\|_ <br> _end_      | `hash.each do |key, value|` <br> `puts "#{key} has value: #{value}"` <br> `end`      |
+| Iterate over pairs | Iterate over all the pairs in the hash                               | _hash.each_pair do \|key, value\|_ <br> _end_ | `hash.each_pair do |key, value|` <br> `puts "#{key} has value: #{value}"` <br> `end` |
```
Comment (hharen) [inline cheat_sheets/hashes.md:13]: They didn't have Enumerables yet, just sayin'

File: cheat_sheets/integers-floats-strings.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+| -------------------- | ------------------------------------------------------- | ------- | ---------------------------------------------------------------------------------------------- |
+| Integer literal      | A whole number                                          |         | `3` <br> `-552`                                                                                |
+| Float literal        | A decimal number                                        |         | `42.23` <br> `-0.133742`                                                                       |
+| Addition             |                                                         | _a + b_ | `5.2 + 6.34 # => 11.54`                                                                        |
+| Subtraction          |                                                         | _a - b_ | `2.59 - 4.89 # => -2.3`                                                                        |
+| Multiplication       |                                                         | _a * b_ | `5 * 3.7 # => 18.5`                                                                            |
```
[the remark this answers is not in the document]
Comment (nethad) [reply cheat_sheets/integers-floats-strings.md:12]: Yeah or we just have 1 point for "addition, subtraction, multiplication" and show 3 examples?

File: cheat_sheets/arrays.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+---
+layout: cheat_sheet
+topic: Arrays
+---
+
+| Name           | Description                                                            | Syntax                                    | Examples                                                |
+| -------------- | ---------------------------------------------------------------------- | ----------------------------------------- | ------------------------------------------------------- |
+| Array literal  | Creates a new array                                                    | _[item1, item2, ...]_                     | `my_array = [1, 2, 3]`                                  |
```
[the remark this answers is not in the document]
Comment (nethad) [reply cheat_sheets/arrays.md:8]: Yeah would also add it

File: cheat_sheets/integers-floats-strings.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (nerdinand) [reply cheat_sheets/integers-floats-strings.md:12]: I'd like to keep them around, for completeness, but yeah, i made it one row now...

File: cheat_sheets/arrays.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (nerdinand) [reply cheat_sheets/arrays.md:8]: ok, done

File: cheat_sheets/hashes.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+| Key existence      | Ask the hash if it has a certain key                                 | _hash.has_key?(key)_                          | `hash.has_key?("key")`                                                               |
+| Value existence    | Ask the hash if it has a certain value                               | _hash.has_value?(value)_                      | `hash.has_value?("xy")`                                                              |
+| All keys           | Get all the keys stored in the hash                                  | _hash.keys_                                   | `hash.keys`                                                                          |
+| All values         | Get all the values stored in the hash                                | _hash.values_                                 | `hash.values`                                                                        |
+| Merge              | Merge two hashes                                                     | _hash.merge(other_hash)_                      | `hash.merge({"a_key" => 23})`                                                        |
+
```
[the remark this answers is not in the document]
Comment (nerdinand) [reply cheat_sheets/hashes.md:20]: cool, done!

File: cheat_sheets/hashes.md (hunk 1 above)
Comment (nerdinand) [reply cheat_sheets/hashes.md:13]: Yeah, i think i'll leave it in there still, for completeness...

**Pull request merged**